### DISCUSSION TITLE
Can I hightlight words?

### DISCUSSION THREAD

**Discussion by alissoncorsair** [0x7c13/Notepads#1015] [Q&A] [answered]
I downloaded it because it looks an amazing app and it seems to be fast, but I also would like to be able to highlight some words to make it clear when I make notes

**Comment by 0x7c13** [comment, marked as answer]
What you are asking for is a "Rich" edit feature which is not possible thru .txt files. You can save your files as .md and use markdown to write notes or you should try out other rich-edit note taking apps. Notepads is a plain text editor.

**Reply by alissoncorsair** [reply]
oops, didn't know that... thank you!
